docs/pages/project/contributing/contributing-error.md

```diff
@@ -145,7 +145,7 @@ func main() {
 		DebugLevel: true,
 	})
 	if err != nil {
-		fmt.Println(err)
+		logrus.Error(err)
```

Comment:
We're looking for MeshKit logger, not logrus, right?

Reply:
@singh1203 will you offer some direction?

Reply:
Yes, @Aviral0702 we are in motion to get rid of logrus with Meshkit logger.